hs_swat_modelinstance/templates/includes/application_scripts.html

```diff
@@ -0,0 +1,17 @@
+{% if debug %}
+<script type="text/javascript" src='{{ STATIC_URL }}js/underscore.js'></script>
+<script type="text/javascript" src='{{ STATIC_URL }}js/json2.js'></script>
+<script type="text/javascript" src='{{ STATIC_URL }}js/backbone.js'></script>
+<script type="text/javascript" src='{{ STATIC_URL }}js/backbone.marionette.js'></script>
+<script type="text/javascript" src='{{ STATIC_URL }}js/backbone-relational.js'></script>
+<script type="text/javascript" src='{{ STATIC_URL }}js/moment-min.js'></script>
+<script type="text/javascript" src='{{ STATIC_URL }}js/jqcsrf.js'></script>
+{% else %}
+<script type="text/javascript" src='{{ STATIC_URL }}js/underscore-min.js'></script>
+<script type="text/javascript" src='{{ STATIC_URL }}js/json2.js'></script>
```

Comment:
@ironfroggy this html (/templates/includes/application_scripts.html), I did not create it. Maybe it was included in the basic files for create new resources. Also I took a look on the other implemented resources and I found this file exist. Do you recommend to remove it totally?